prExplorer/client-app/src/app/common/form/CustomSelectInput.tsx

```diff
@@ -0,0 +1,26 @@
+import { useField } from 'formik';
+import React from 'react';
+import { Form, Label, Select } from 'semantic-ui-react';
+
+interface CustomSelectInputProps {
+    placeholder: string;
+    name: string;
+    options: any;
+    label?: string;
+}
+
+export default function CustomSelectInput(customSelectInputProps: CustomSelectInputProps) {
+    const [field, meta, helpers] = useField(customSelectInputProps.name);
+    return (
+        <Form.Field error={meta.touched && !!meta.error}>
```

Comment:
@aliyorulmazdev are u sure you use Prettier in VS Code? Your code does not seem to be formatted

Reply:
I will check that